bob-apps/Cargo.toml

```diff
@@ -43,6 +43,7 @@ tower = "0.4"
 rand = "0.8"
 tower-service = "0.3"
 log4rs-logstash = { version = "0.1.0", features = ["rustls"] }
+reqwest = "0.11"
```

Comment:
`reqwest` depends on `openssl` and that leads to compile error. You need to disable `tls` support (https://github.com/seanmonstar/reqwest/issues/495): `reqwest = { version = "0.11", default-features = false }`